### DISCUSSION TITLE
No puedo vincular las referencias del texto con la bibliografia al convertir de XML Jats hacia HTML

### DISCUSSION THREAD

**Discussion by gsuruguay** [jgm/pandoc#10475] [Q&A]
Hola!

Intento generar un html partiendo desde un XML JATS que ya está validado y correcto, pero al crearme la sección de referencias, no logro dar con un archivo correcto de .csl que me las vincule al convertir.

Estoy usando el siguiente pero me da errores

<?xml version="1.0" encoding="utf-8"?>
<style xmlns="http://purl.org/net/xbiblio/csl" version="1.0">
  <info>
    <title>Vancouver with Links</title>
    <id>http://www.zotero.org/styles/vancouver-with-links</id>
    <link href="http://www.zotero.org/styles/vancouver-with-links" rel="self"/>
    <author>
      <name>Adapted Vancouver Style</name>
    </author>
    <updated>2024-01-01T00:00:00+00:00</updated>
  </info>
  <citation>
    <layout prefix="[" suffix="]" delimiter=",">
      <text variable="citation-number">
        <link macro="citation-link"/>
      </text>
    </layout>
  </citation>
  <bibliography>
    <layout>
      <text variable="citation-number" prefix="<a id='ref-" suffix="'></a>"/>
      <text variable="author" suffix=". "/>
      <text variable="title" suffix=". "/>
      <group delimiter=" ">
        <text variable="container-title"/>
        <text variable="volume"/>
        <text variable="issue" prefix="(" suffix=")"/>
        <text variable="page"/>
      </group>
      <text variable="issued" prefix=" "/>
      <text variable="DOI" prefix=" DOI: ">
        <link macro="doi-link"/>
      </text>
    </layout>
  </bibliography>
  <macro name="citation-link">
    <text variable="citation-number" prefix="<a href='#ref-" suffix="'></a>"/>
  </macro>
  <macro name="doi-link">
    <text variable="DOI" prefix="<a href='https://doi.org/" suffix="'></a>"/>
  </macro>
</style>

Suelo obtener el siguiente error al ejecutar el comando pandoc -s -f jats -t html5 --citeproc --csl=nuevo.csl --embed-resources --css=estilos.css -o salida.html origen.xml 

Error obtenido:
CiteprocXMLError: ParseError {errorContexts = ["open tag","'>'"], errorMessage = "Failed reading: satisfy", errorPosition = 20:15 (611)}

**Comment by badumont** [comment]
Sorry, I can read but not write Spanish. You should not write HTML code in the prefix and suffix attributes. Pandoc has the link-citations and link-bibliography metadata to do what you seem to try to achieve.
